reference/format/dashboard.rst

```diff
@@ -0,0 +1,189 @@
+|           |   * `route`_               |
+|           |   * `route_parameters`_    |
+|           |                            |
+|           | * `widgets_configuration`_ |
++-----------+----------------------------+
+
+The ``dashboard.yml`` file is used to configure dashboards and widgets that are rendered on a
+dashboard:
+
+.. code-block:: yaml
+    :linenos:
+
+    # src/Acme/DemoBundle/Resources/config/dashboard.yml
+    oro_dashboard_config:
+        widgets:
+            # a "my_calendar" widget displaying the user's calendar on the dashboard
+            my_calendar:
+                label:       oro.calendar.my_calendar_widget_title
+                route:       oro_calendar_dashboard_my_calendar
+                acl:         oro_calendar_view
+                description: oro.calendar.my_calendar_widget_description
+                icon:        bundles/orocalendar/img/my_calendar.png
+
+        dashboards:
+            # a dashboard named "main" using the given template
+            main:
+                twig: OroDashboardBundle:Index:default.html.twig
+
+        widgets_configuration:
+            # a "title" option for all dashboard widgets
+            title:
+                type: oro_type_widget_title
+                options:
+                    label: oro.dashboard.title.label
+                    required: false
+
+``dashboards``
+--------------
+
+**type**: ``map``
+
+The ``dashboards`` key is used to configure options for dashboards. The key is the name of the
+dashboard and the options are the values. Currently, there is only one supported option:
+
+.. _reference-format-dashboard-twig:
+
+``twig``
+~~~~~~~~
+
+**type**: ``string``
+
+The Twig template used to render the dashboard. ``OroDashboardBundle:Index:default.html.twig`` is
+used when no value is configured.
+
+``widgets``
+-----------
+
+**type**: ``map``
+
+The keys are strings that each uniquely identiy a dashboard widget. The value again is a map that
+holds the configuration of the widget for which the following options can be used:
+
+``acl``
+~~~~~~~
+
+**type**: ``string``
+
+The given access control list is evaluated to decide whether or not a user is allowed to see the
+widget.
+
+``configuration``
+~~~~~~~~~~~~~~~~~
+
+**type**: ``map``
+
+You can define an arbitrary set of configuration options the user can use to configure the widget
+(for example to customize the way it is rendered, its behavior, etc.). Each key of the map
+represents the name of a configuration option, its values are maps that can use the following keys:
+
+``options`` (**type**: ``map``)
+
+    A map of options and their values configuring the Symfony form type (see the ``type`` option
+    below). You can find a complete reference of available options for Symfony's built-in form
+    types `in the Symfony documentation`_.
+
+``show_on_widget`` (**type**: ``boolean``)
+
+    If set to ``true``, the user can edit the option at the bottom of the widget. Otherwise, the
```

Comment:
This description is wrong. User can edit options only by clicking on the wheel icon. If this parameter is set to true, the value of this option will be shown at the bottom of widget. For example, at http://demo.orocrm.com/desktop/ page, we have Customers Activity widget. At the bottom of this widget you can see current daterange for this widget. This was done with `show_on_widget` parameter of this option. But you cannot edit this value directly on the widget.